src/Media/MediaLeft.js

```diff
@@ -0,0 +1,31 @@
+import React, {Component, PropTypes} from 'react';
+
+export default class MediaLeft extends Component {
+  static propTypes = {
+    /**
+     * The media will link to the address given here
+     */
+    href: PropTypes.string,
+    /**
+     * Align the media to one of these positions
+     */
+    align: PropTypes.oneOf(['top', 'middle', 'bottom'])
+  }
+
+  static defaultProps = {
+    href: '#',
+    align: 'top'
+  }
+
+  render() {
+    const {href, align, children, className} = this.props;
+
+    return (
+      <div className={`media-left media-${align} ${className}`}>
+        <a href={href}>
```

Comment:
We don't need this – if the user needs a link, he or she can pass in an `<a>` as children to this component.